src/main/kotlin/ink/abb/pogo/scraper/tasks/ProcessPokestops.kt

```diff
@@ -50,6 +50,12 @@ class ProcessPokestops(var pokestops: MutableCollection<Pokestop>) : Task {
         if (startPokestop == null)
             startPokestop = sortedPokestops.first()
 
+        if (ctx.lootedPokestops.get() > settings.pokestopThreshold) {
+            Thread.sleep(3600 * 12000)
```

Comment:
@jabbink I can reassign the settings.catchPokemon value, but not the settings.lootPokestop value.